keylime/tpm/tpm_abstract.py

```diff
@@ -230,13 +230,16 @@ def __check_ima(self, agent_id, pcrval, ima_measurement_list, allowlist, ima_key
         logger.debug(f"IMA measurement list of agent {agent_id} validated")
         return True
 
-    def check_pcrs(self, agent_id, tpm_policy, pcrs, data, virtual, ima_measurement_list, allowlist, ima_keyring):
+    def check_pcrs(self, agent_id, tpm_policy, pcrs, data, virtual, ima_measurement_list, allowlist, ima_keyring, mb_measurement_list, mb_intended_state):
         try:
             tpm_policy_ = ast.literal_eval(tpm_policy)
         except ValueError:
             tpm_policy_ = {}
         pcr_allowlist = tpm_policy_.copy()
 
+        if mb_measurement_list or mb_intended_state :
+            logger.info("Measured boot information received, but for now it will not be processed")
```

Comment:
This could perhaps steer the user more towards the reason why. If I saw this message I would think 'ok, when will it be processed and why?'.

Reply:
the message will disappear when PR2 shows up. Probably should have marked this as "temporary"